db/seeds.rb

```diff
@@ -6,31 +6,68 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+# puts "seeds.rb"
+
 # the '99' is to make sure we leave space for the migrated divisions
-Division.find_or_create_by({id:99, internal_name: Division.root_internal_name, name:'Root Division'})
+Division.find_or_create_by(id: 99, internal_name: Division.root_internal_name, name:'Root Division')
 # note, divisions created within the new system will start at 101
 Division.connection.execute("select setval('divisions_id_seq', greatest((select max(id) from divisions), 100))")
 
-# note, these hardcoded id's are needed to match the migrated data
-Language.find_or_create_by({id:1,name:'English',code:'EN'})
-Language.find_or_create_by({id:2,name:'Español',code:'ES'})
-Language.find_or_create_by({id:3,name:'Français',code:'FR'})
+
+Language.find_or_create_by(id: 1, name: 'English', code: 'EN')
+Language.find_or_create_by(id: 2, name: 'Español', code: 'ES')
+Language.find_or_create_by(id: 3, name: 'Français', code: 'FR')
```

Comment:
I touched on this in the thread above.  My inclination is to only keep the English (and maybe Spanish) Language rows in the seeds.rb and to remove all of the Country and Currency entries to a migration specific file.  We could theoretically introduce a `migration_id` field on these tables, but I don't the complexity there is really justified, especially with most of that data removed from seeds.rb.